packages/back-end/src/util/features.ts

```diff
@@ -346,6 +356,29 @@ export function getFeatureDefinition({
     ? revision.rules?.[environment] ?? settings.rules
     : settings.rules;
 
+  // If the feature has a holdout and it's enabled for the environment, add holdout as a
+  // pseudo force rule with a prerequisite condition. The environment being enabled is
+  // already checked in the getAllPayloadHoldouts function.
+  const holdoutRule: FeatureDefinitionRuleWithHoldout[] =
+    feature.holdout &&
+    holdoutsMap &&
+    holdoutsMap.get(feature.holdout.id)?.environmentSettings?.[environment]
+      ?.enabled
+      ? [
+          {
+            id: `fr_${uuidv4()}`,
+            parentConditions: [
+              {
+                id: getHoldoutFeatureDefId(feature.holdout.id),
+                condition: { value: "holdoutcontrol" },
+              },
+            ],
+            force: getJSONValue(feature.valueType, feature.holdout.value),
+            holdoutId: feature.holdout.id, // Used to do holdout project filtering. Scrubbed in the getFeatureDefinitionsResponse function.
```

Comment:
We might not need this.  The pre-requisite scrubbing should remove this rule already based on the parentCondition always being false (holdout feature doesn't exist)